src/Headers/Hydrodynamics.h

```diff
@@ -50,7 +51,7 @@ class EOS;
 
 class SimulationBase;
 
-static const FLOAT ghost_range = 2.5;
+static const FLOAT ghost_range = 3.0; //2.5;
```

Comment:
This issue with a small change in the error for the sedov test has appeared a few times and then dissappeared. It could be a ghost range issue, but I'd be surprised. I've seen it occur occaisonally, it was the origin for the failure in this [commit](https://github.com/gandalfcode/gandalf/pull/157/commits/069a6ea907bec4ca3206ab622da787ebcd80ef87).